students/andykwon/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,75 @@
+def exchange_first_and_last(seq):
+    print("this is exchange_first_and_last")
+
+    return seq[-1:] + seq[1:-1] + seq[:1]
+
+    # first = seq[:1]
+    # last = seq[-1:]
+    # middle = seq[1:-1]
+
+    # return first + last + middle
+
+def every_other_removed(seq):
+    print("this is every_other_removed")
+
+    return seq[::2]
+
+def remove_first_last_four_and_every_other(seq):
+    print("this is remove_first_last_four_and_every_other")
+
+    seq = seq[4:]
+    seq = seq[:-4]
+    seq = seq[::2]
+
+    return seq
+
+def elements_reversed_slicing(seq):
+    print("this is elements_reversed_slicing")
+
+    return seq[::-1]
+
+def reorder_middle_last_first_third(seq):
+    print("this is reorder_middle_last_first_third")
+
+    first_third = seq[:int(len(seq)/3)]
```

Comment:
`len(seq) // 3`

will return an int saving that call.

and good idea to store it, so you don't need to call len() a whole bunch of times.